src/main/resources/fabric.mod.json

```diff
@@ -22,7 +22,7 @@
 	],
 
 	"depends": {
-		"fabricloader": ">=0.7.4",
-		"minecraft": "1.16.x"
+		"fabricloader": ">=0.11.3",
+		"minecraft": "1.17.x"
```

Comment:
Could you make this compatible with both 1.16 and 1.17?

Reply:
Ok, I will do that.

Reply:
done